cheri_compressed_cap_common.h

```diff
@@ -207,10 +214,23 @@ static inline _cc_bounds_bits _cc_N(extract_bounds_bits)(_cc_addr_t pesbt) {
         // Do not offset by 1! We also need to encode E=0 even with IE
         // Also allow nonsense values over 64 - BWidth + 2: this is expected by sail-generated tests
         // E = MIN(64 - BWidth + 2, E);
+#ifdef IS_MORELLO
```

Comment:
You have a bunch of these in need of fixing

Reply:
Another result of the bad rebase. Gone now.